WordPress/src/main/res/drawable/bg_colorful_dots.xml

```diff
@@ -0,0 +1,21 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="516dp"
+    android:height="157dp"
+    android:viewportWidth="516"
+    android:viewportHeight="157">
+  <path
+      android:pathData="M67.613,140.956m-4.584,0a4.584,4.584 0,1 1,9.168 0a4.584,4.584 0,1 1,-9.168 0"
+      android:fillColor="#1ED15A"/>
```

Comment:
@ashiagr wdyt about replacing with color resource references instead of hex values?

Reply:
Replaced here: 463e54e